src/main/resources/edu/cornell/kfs/sys/businessobject/datadictionary/ISOCountry.xml

```diff
@@ -0,0 +1,169 @@
+          parent="AttributeDefinition"
+          p:description="The ISO code uniquely identifying the country."
+          p:forceUppercase="true"
+          p:label="ISO Country Code"
+          p:maxLength="2"
+          p:name="code"
+          p:required="true"
+          p:shortLabel="ISO Country Code"
+          p:summary="ISO Postal Country Code"
+    >
+        <property name="control">
+            <bean parent="TextControlDefinition" p:size="2"/>
+        </property>
+        <property name="validationPattern">
+            <bean parent="AlphaNumericValidationPattern"/>
+        </property>
+    </bean>
+
+    <bean id="ISOCountry-alternateCode" parent="ISOCountry-alternateCode-parentBean"/>
+    <bean abstract="true"
+          id="ISOCountry-alternateCode-parentBean"
+          parent="AttributeDefinition"
+          p:description="The alternate ISO code uniquely identifying a country."
+          p:forceUppercase="true"
+          p:label="Alternate ISO Country Code"
+          p:maxLength="3"
+          p:name="alternateCode"
+          p:shortLabel="Alt ISO Country Code"
+          p:summary="Alternate ISO Postal Country Code"
+    >
+        <property name="control">
+            <bean parent="TextControlDefinition" p:size="3"/>
+        </property>
+        <property name="validationPattern">
+            <bean parent="AlphaNumericValidationPattern"/>
+        </property>
+    </bean>
+
+    <bean id="ISOCountry-name" parent="ISOCountry-name-parentBean"/>
+    <bean abstract="true"
+          id="ISOCountry-name-parentBean"
+          parent="AttributeDefinition"
+          p:description="ISO Postal Country Name"
+          p:label="ISO Country Name"
+          p:maxLength="255"
+          p:name="name"
+          p:required="true"
+          p:shortLabel="ISO Country Name"
+          p:summary="ISO Postal Country Name"
+    >
+        <property name="control">
+            <bean parent="TextControlDefinition" p:size="50"/>
+        </property>
+    </bean>
+
+    <bean id="ISOCountry-active" parent="ISOCountry-active-parentBean"/>
+    <bean abstract="true"
+          id="ISOCountry-active-parentBean"
+          parent="GenericAttributes-activeIndicator"
+          p:name="active"
+    />
+
+    <!-- Business Object Inquiry Definition -->
+    <bean id="ISOCountry-sectionDefinition" parent="ISOCountry-sectionDefinition-parentBean"/>
+    <bean abstract="true"
+          id="ISOCountry-sectionDefinition-parentBean"
+          parent="sectionDefinition"
+    >
+        <property name="fields">
+            <list>
+                <ref bean="ISOCountry-code"/>
+                <ref bean="ISOCountry-name"/>
+                <ref bean="ISOCountry-alternateCode"/>
+                <ref bean="ISOCountry-active"/>
+            </list>
+        </property>
+    </bean>
+
+    <!-- Business Object Lookup Definition -->
+    <bean id="ISOCountry-inquiryDefinition" parent="ISOCountry-inquiryDefinition-parentBean"/>
```

Comment:
This comment is out of place. The comment should be moved over to the lookup bean instead of the inquiry bean.

Reply:
moved